### ISSUE TITLE
The build directory should be gitignored 

### ISSUE THREAD

**Issue by ChristopherChudzicki** [unconed/mathbox#21]
Current situation:
- the `build/` directory contains build mathbox (a bundle for the web, and a esm version transpiled from typescript)
- the build directory in version control is NOT used by npm** because the `prepack` script rebuilds.

It would be nice to not have to manually update / deal with the build directory.
- **Option 1**:  remove the build directory from version control; reference some other build version (e.g., from a CDN) in the examples 
- **Option 2**: keep the build directory in VC, but somehow automatically build-and-commit when PRs are merged to master. 

Are there other options?

**Comment by ChristopherChudzicki** [comment]
Option 1 seems simpler, so I lean toward that approach. The only disadvantage I can think of is that if all the examples use a CDN, it makes it slightly harder to manually test the examples for regressions. But... we can work around that, e.g., with a find-and-replace for temporary manual testing. Or better... include the examples in automated tests (like I started with `curve`). Anyway,  I think that downside is acceptable and we can improve the manual regression testing situation later.

**Comment by sritchie** [comment]
I think that's great; including the examples in tests is the right long term move.

I bet we can get Github Actions to build the bundles on each PR and make the artifact available.

I also would love your thoughts on WHAT these bundles should be. I modified @unconed's original strategy of offering a monster bundle with all of three.js in it, and a mathbox-core with no dependencies. That is way easier for people to consume, but makes it not possible to pick a three.js version on your own.

At the same time, I have found issues with requirejs, or d3.require, not understanding that THREE needs to be installed as global THREE for OrbitControls etc to initialize.

So, this strategy is good, and I'm not sure of the best long-term path.
